packages/react-heartwood-components/src/components/RecordSelectionList/RecordSelectionList.js

```diff
@@ -310,60 +316,100 @@ export default class RecordSelectionList extends Component<
 				this.list.forceUpdateGrid()
 			}
 		}
+		const isListShort =
+			totalRecordCount &&
+			loadedRecords &&
+			loadedRecords.length === totalRecordCount
```

Comment:
@jtini should this be `<=` in case records are filtered out for some reason but totalRecordCount remains?

Reply:
@brandonminch Good point, but I don't think that can happen here. If I make that change, we'll get the non-virtual list 100% of the time. I think that `totalRecordCount` needs to be a reflection of the exact (filtered) request, not the actual total.